kbase-extension/static/kbase/js/common/cellComponents/actionButtons.js

```diff
@@ -67,7 +67,7 @@ define(['common/html'], (html) => {
                 });
             });
             bus.on('actionButton', (message) => {
-                const action = message.data;
+                const action = message.data.action;
```

Comment:
The "action" being run was the object `{action: 'someString'}` when it should be just `'someString'`.